js/index.js

```diff
@@ -1,6 +1,96 @@
 // Iteration 1: Names and Input
-//
-// Iteration 2: Conditionals
+const hacker1 = "Gaston";
+console.log(`The driver's name is ${hacker1}`);
+
+const hacker2 = "Leandro";
+console.log(`The navigator's name is ${hacker2}`);
 
+// Iteration 2: Conditionals
+if (hacker1.length === hacker2.length) {
+  console.log(`Wow, you both have equally long names, ${hacker1.length} characters`);
+} else if (hacker1.length < hacker2.length) {
+  console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters`);
+} else {
+  console.log(`The driver has the longest name, it has ${hacker1.length} characters`);
+}
 
 // Iteration 3: Loops
+// 3.1
+
+let spacedCapStr = "";
+for (let i = 0; i < hacker1.length; i++) {
+  spacedCapStr += hacker1[i].toUpperCase() + " ";
+}
+
+console.log(`"${spacedCapStr.slice(0, -1)}"`);
```

Comment:
Perfecto!!!, bien controlado!, sabrias decirme otras formas para obtener el mismo resultado? (esto va por fuera del ejercicio, porque literalmente esta perfecto)

Reply:
Estas serían dos opciones mas que pude realizar: 
`// Option A
const spacedHacker1 = []
for (let letter of hacker1) {
  spacedHacker1.push(letter.toUpperCase())
}
const spacedHacker1Joined = spacedHacker1.join(" ")

console.log(spacedHacker1Joined);

// Option B
let spacedHackerWhile = ''
let i = 0
let stopWhile = false

while (!stopWhile && i < hacker1.length) {
  for (let letter of hacker1) {
    spacedHackerWhile = spacedHackerWhile.concat(letter.toUpperCase() + " ")
  }
  stopWhile = true
}
spacedHackerWhile = spacedHackerWhile.trim()

console.log(spacedHackerWhile);`